scripts/ci/attach_ebs_cache.sh

```diff
@@ -123,12 +124,12 @@ done
 # We are expecting the device to come up as /dev/nvme1n1, but include generic code from
 # https://github.com/slavivanov/ec2-spotter/blob/master/ec2spotter-remount-root
 while true; do
-    if lsblk /dev/nvme1n1; then
+    if sudo lsblk /dev/nvme1n1; then
```

Comment:
maybe just run this script with sudo?

Reply:
sudo had the annoying thing that it doesn't read env vars

Reply:
ah I could have just used -E